root/area/merge.tt

```diff
@@ -4,16 +4,29 @@
     <p>[% l('You are about to merge the following areas into a single area. Please
              select the area which you would like other areas to be merged into:') %]</p>
     <form action="[% c.req.uri %]" method="post">
-        <ul>
```

Comment:
Area and artist are the only merges currently using an `<ul>` for some reason. Changed this to a table for consistency (and to reuse the same macro).